migrations/V64__Add_IntakeID_To_Accessibility_Request.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE accessibility_request
+    ADD COLUMN intake_id UUID NOT NULL REFERENCES system_intake (id);
```

Comment:
I agree with what you're saying here in theory, but for this specific project (and considering the size and relative immaturity of the project), having the constraint that we have to have a system that it is located in the same database seems like a better fit. Looser coupling means we have to handle more situations (both in code and at a policy/product level), and I don't think we have the bandwidth to do that right now.

In any case, we can always relax this restriction later on or provide it easy to add a system (which doesn't really need to be a system intake) so that satisfying this requirement isn't a blocker.

Reply:
I thought of another, more immediate, outcome... 
To test the `AccessibilityRequest` functionality, you then need to ensure there is a `system_intake` (in the proper state?) in the database... 
Part of the reason we have the fakey `System` instances available in pre-PROD environments is to make it easier by reducing set up overhead... (In DEV especially, since DB gets blown away on each deploy.)

Reply:
@nelztruss that is correct, but our app is expecting the connection to be there. If we need to invest time in making good test data, then let's do that, rather than sacrifice some of our referential integrity.